express/src/services/lambda-facade/StubLambdaFacade.ts

```diff
@@ -61,14 +65,23 @@ class StubLambdaFacade implements LambdaFacadeInterface {
         serviceId: string,
         selfServiceClientId: string,
         clientId: string,
-        updates: Record<string, string>,
+        updates: Record<string, unknown>,
```

Comment:
```suggestion
        updates: Record<string, string | string[]>;,
```
Use the specific type and the `convertToAttr` SDK method to automatically handle type conversions and avoid type warnings.